src/main/kotlin/src/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,22 @@
+package src.racingcar.domain
+
+open class Car(
+    val name: String,
+    moveCount: Int = 0
+) {
+
+    var moveCount: Int = moveCount
+        protected set
```

Comment:
```suggestion
    var moveCount: Int = moveCount
        private set
```

kotlin에서 protected와 private의 차이는 무엇일까요~?